### ISSUE TITLE
Can't remove popups or clear notifications in the notifications center

### ISSUE THREAD

**Issue by g4rrucho** [phuhl/linux_notification_center#69]
I've tried the command to clear the notifications, with notify-send and notify-send.py, both in the center and the popups. Both don't work ATM for me.
Curious enough, I can turn on and off the buttons highlight, so maybe it's not a problem from notify-send.py.

Might this be a bug? 
OS: arch linux 5.7.11
WM: i3-gaps
Shell: zsh + oh-my-zsh

**Comment by MyriaCore** [comment]
Weird. I assume you installed via AUR? I'm on manjaro and my stuff is working. You shouldn't *need* to install notify-send.py, as the following commands should work:

```sh
# within the notification center
notify-send a -h int:deadd-notification-center:1 \
              -h string:type:clearInCenter

# popups
notify-send a -h int:deadd-notification-center:1 \
              -h string:type:clearPopups
```

And, really, the buttons should *definitely* work, since the same function is called for the buttons as it is for the commands.

**Comment by MyriaCore** [comment]
Try running `deadd-notification-center` in your terminal and see what happens when you type these commands out. 

If you see `clearing in center` and `clearing popups` printed out, that means things work well enough to get to the `deleteInCenter` and `hideAllNotis` functions. The code for those prints is here:

https://github.com/phuhl/linux_notification_center/blob/69b160676064418dd08ebc6b65763f31a649f061/src/NotificationCenter.hs#L251-L259

From how you're describing it, I get the sense that maybe it *is* reaching these properly, but then maybe something's going wrong inside each function. 

This is all assuming that the bug is with deadd, and not with something else. When I ran into #60, I figured it was a bug with the code here, but now I think it's a lightdm / systemd thing.
